habitat-lab/habitat/config/habitat/task/rearrange/pddl/multi_agent_social_nav_solved.yaml

```diff
@@ -0,0 +1,87 @@
+# Version of tidy house that only has two target objects.
+objects:
+  - name: any_targets|0
+    expr_type: movable_entity_type
+  - name: TARGET_any_targets|0
+    expr_type: goal_entity_type
+  - name: any_targets|1
+    expr_type: movable_entity_type
+  - name: TARGET_any_targets|1
+    expr_type: goal_entity_type
+  - name: robot_0
+    expr_type: robot_entity_type
+  - name: robot_1
+    expr_type: robot_entity_type
+
+init:
+  - closed_fridge(fridge_push_point)
+
+goal:
+  expr_type: AND
+  sub_exprs:
+    - at(any_targets|0,TARGET_any_targets|0)
+    - at(any_targets|1,TARGET_any_targets|1)
+    - not_holding(robot_0)
+    - not_holding(robot_1)
+
+stage_goals:
+  stage_1:
+    expr_type: AND
+    sub_exprs:
+      - at(any_targets|0,TARGET_any_targets|0)
+  stage_2:
+    expr_type: AND
+    sub_exprs:
+      - at(any_targets|0,TARGET_any_targets|0)
+      - at(any_targets|1,TARGET_any_targets|1)
+
+solution:
+  - nav(any_targets|0, robot_1)
+  - pick(any_targets|0, robot_1)
+  - nav(TARGET_any_targets|0, robot_1)
+  - place(any_targets|0,TARGET_any_targets|0, robot_1)
+  - nav(any_targets|1, robot_1)
+  - pick(any_targets|1, robot_1)
+  - nav(TARGET_any_targets|1, robot_1)
+  - place(any_targets|1,TARGET_any_targets|1, robot_1)
+  - nav(robot_1, robot_0)
```

Comment:
I think we should have a more elegant solution without having to copy the action in the config. Maybe you can create a new "follow" HL policy that does this behavior without having it to specify it in the PDDL problem file.

Reply:
@jimmytyyang , maybe it is a good place to try the PlannerHighLevelPolicy. You can create a follow policy that subclasses the planner but [here](https://github.com/facebookresearch/habitat-lab/blob/SIRo_ma_fixes6/habitat-baselines/habitat_baselines/rl/hrl/hl/planner_policy.py#L189) outputs always a oracle navigation action

Reply:
Hi all, I might need to take some time to look at this, but giving the priority is training the policy, I will fix this later